integration-tests/src/test/java/org/apache/druid/tests/indexer/AbstractITRealtimeIndexTaskTest.java

```diff
@@ -97,6 +97,8 @@ void doTest()
       LOG.info("indexerSpec: [%s]\n", task);
       taskID = indexer.submitTask(task);
 
+      TimeUnit.SECONDS.sleep(5);
```

Comment:
Could you please include a comment about why this is happening?

Reply:
Done

Reply:
This integration test is so gross. No wonder it's flaky. But, okay, sure, fine.